src/Niekert/DiscordMCPE/Events/EventListener.php

```diff
@@ -32,7 +32,7 @@ public function onJoin(PlayerJoinEvent $event)
     {
         $playername = $event->getPlayer()->getDisplayName();
         if ($this->main->joinopt !== '0') {
-            if ($this->main->pp !== null) {
+            if ($this->main->pp == true) {
```

Comment:
I just realised that is really wrong.

Previously, I was checking if the `pureperms` in the config didn't equal to null... which meant, you could put anything there and, if it didn't equal to null, it would continue on with that statement. I don't know why I thought that statement would be correct... I might've been thinking that it was checking if the plugin equaled to null (pureperms).